Tests/Unit/DisplayerTests.swift

```diff
@@ -0,0 +1,21 @@
+import Quick
+import Nimble
+@testable import MiniApp
+
+class DisplayerTests: QuickSpec {
+
+    override func spec() {
+        describe("get mini app view") {
+            context("when valid URL of a mini app is passed") {
```

Comment:
What happens when the URL is invalid?

Reply:
I need to use failable initializers then. Working on it

Reply:
Done